.gitlab-ci.yml

```diff
@@ -30,6 +31,7 @@ build:image:
   script:
     - bin/build
     - mkdir -p docker/images
+    - ARNOLD_IMAGE="arnold:$(tr -d '\n' < VERSION)"
     - docker save -o docker/images/arnold.tar ${ARNOLD_IMAGE}
   # Store the built as an artifact that can be reused in other jobs
   artifacts:
```

Comment:
BTW, I think these changes should not be part of this PR (out of scope).

Reply:
You're right.
I'm open an issue for sync circleCI and gitlab-ci files.